app/models/utils.rb

```diff
@@ -0,0 +1,35 @@
+module Utils
```

Comment:
Jak już tworzymy Utils dla tej usługi, to dodaj proszę ten kawałek kodu:
```
  # extend this Utils with Scalarm::ServiceCore::Utils module methods
  class << self
    Scalarm::ServiceCore::Utils.singleton_methods.each do |m|
      define_method m, Scalarm::ServiceCore::Utils.method(m).to_proc
    end
  end
```
Dzięki temu to będzie rozszerzenie Utilsów z ServiceCore